api/allIncidents/incidentsModel.js

```diff
@@ -114,7 +113,12 @@ function getLastID() {
  * Ideally google maps API can take city/state and populate lat/long on backend.
  */
 async function createIncident(incident) {
+  const newMax = await db('force_ranks').max('incident_id').first();
+  const { max } = newMax;
+  const newIncidentId = max + 1;
+
   const newIncident = {
```

Comment:
For some reason it appears the Table was not create with auto-increment. This may have been something done by previous DS teams. This function pulls the highest incident report and adds one to create a new incident. If incident_id is null it will error out.